plugin.xml

```diff
@@ -163,6 +163,7 @@
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
       <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
           android:label="@string/app_name"
+          android:launchMode="singleTop"
```

Comment:
Sorry. Forgot to remove that after testing. Will update the commit to remove that.